packages/perseus/src/widget-ai-utils/dropdown/prompt-utils.ts

```diff
@@ -1,22 +1,19 @@
-import type {WidgetType} from "../../prompt-types";
 import type {PerseusDropdownUserInput} from "../../validation.types";
 import type dropdown from "../../widgets/dropdown/dropdown";
-import type {PropsFor} from "@khanacademy/wonder-blocks-core";
-
-type WidgetProps = PropsFor<typeof dropdown.widget>;
+import type React from "react";
 
 export type DropdownPromptJSON = {
-    type: WidgetType;
+    type: "dropdown";
     options: {
-        items: WidgetProps["choices"];
+        items: ReadonlyArray<string>;
     };
     userInput: {
-        selectedIndex: PerseusDropdownUserInput["value"];
+        selectedIndex: number;
     };
 };
```

Comment:
Nice. Looking through this I like how we aren't using any Perseus types for these output types. Thanks for making this change.